config/prow/cluster/build_kubernetes-external-secrets_customresource.yaml

```diff
@@ -13,3 +13,16 @@ spec:
   - key: azure-cred
     name: credentials
     version: latest
+---
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: snyk-token
+  namespace: test-pods
+spec:
+  backendType: gcpSecretsManager
+  projectId: k8s-infra-prow-build-trusted # gsm project id
+  data:
+  - key: snyk-token  # name of the GCP secret
+    name: SNYK_TOKEN  # key name in the k8s secret
+    version: latest
```

Comment:
This is going to need to be redone as a PR to https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources/test-pods-externalsecrets.yaml

Let me check on the job definition PR